### ISSUE TITLE
Fix stats

### ISSUE THREAD

**Issue by LeagueSandboxBot** [LeagueSandbox/GameServer#575]
There're some *wrong* things with damage calculations (i.e. turret damage varies on *your* ad). Fix them all.

---
Beep, boop, I'm [a bot](https://github.com/LeagueSandbox/IssueBot)! This issue was created by `@Furkan_S` in `#game-server`.

**Comment by DannyDelic** [comment]
I will assign myself to this one :)

**Comment by danil179** [comment]
Verify this issue. I think it is outdated.

**Comment by GamesGrandpa** [comment]
I tried giving myself more bonus AD with 6 infnity edges, but there was no change in damage received from turrets. I tried more base AD by leveling myself up, but the post mitigation and pre mitigation damage remains consistent. To be safe I also tried AP items, but with no success in replicating this issue.
The attached file contains my calculations, including champion info, turret info, and the source for the formula used.
[dmg variance.txt](https://github.com/LeagueSandbox/GameServer/files/8457640/dmg.variance.txt)
